src/locales/zh.json

```diff
@@ -901,7 +901,7 @@
       "address_conversion_description": "CKB2021 地址格式已从 bech32 变为 bech32m，我们建议开发人员在 dApp 中使用新格式。此工具可帮助您确定地址是已废弃还是新地址。",
       "address_rfc": "CKB 地址格式 RFC",
       "address_to_script": "Address to Script",
-      "script_to_address": "Script to Address",
```

Comment:
`script_to_address` is used at https://github.com/Magickbase/ckb-explorer-frontend/pull/316/files#diff-79c7cf7369a99998e79acd2e13e64307c13c662e0a5553226c211195930c5f0dR33